.github/workflows/swift.yml

```diff
@@ -21,6 +21,6 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - name: Set Xcode Version
-        run: sudo xcode-select -s /Applications/Xcode_12.app
```

Comment:
You will need to switch this back to Xcode 12, 13 isn't available.

Current software: https://github.com/actions/virtual-environments/blob/macOS-10.15/20210801.1/images/macos/macos-10.15-Readme.md

You can check the issues in the actions repo for info on Xcode 13

Reply:
If you select 12.4, you will need to update the simulator version:

https://github.com/carekit-apple/CareKit/blob/59146eb13b3812db9872ce9b7c8574562b6aba9e/.github/workflows/swift.yml#L18

You should confirm with @erik-apple about these updates

Reply:
I recommend switching this back to Xcode 12 as 12.4 as those changes won't enable the build of your PR (async/await). The reason I recommend this is say for example, there are other reasons the CareKit team has not to jump to a higher version of Xcode, you don't want that getting in the way of your PR being accepted.

 I opened #618 to allow the CareKit team assess the upgrade to Xcode 12.4 separately from this PR.